supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -1,3 +1,37 @@
+[discrete]
+[[aarch64]]
+==== image:images/yes.png[yes] AArch64 (noun)
+*Description*: A 64-bit version of the ARM architecture. Use this term when referring to operating systems and server instances, for example RHEL, Fedora, CoreOS, and other Linux distributions. Use this format in general cases when referring to system architecture. 
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
+
+Example:
+
+_When running RHEL with an AArch64 system, run the following command:_
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:aarch64[aarch64]
```

Comment:
*See also*: xref:aarch64[_aarch64]

Reply:
Ah! Thank you, my apologies, I keep missing these small things